framework/wazuh/core/utils.py

```diff
@@ -1237,7 +1237,7 @@ def __init__(self, offset, limit, table, sort, search, select, query, fields, de
             r'(\()?' +  # A ( character.
             r'([\w.]+)' +  # Field name: name of the field to look on DB
             '([' + ''.join(self.query_operators.keys()) + "]{1,2})" +  # Operator: looks for =, !=, <, > or ~.
-            r"((?:[\[\]\w _\-.:\\/']+(?:\([\[\]\w _\-.:\\/']*\))*)+)" +  # Value: A string.
+            r"((?:[\[\]\w _\-.:\\/'{}]+(?:\([\[\]\w _\-.:\\/'{}]*\))*)+)" +  # Value: A string.
```

Comment:
When merging into 4.4/master, we need to take into account that the regex is

https://github.com/wazuh/wazuh/blob/b4c9ab6680d318b1ae20d73b72d6d5450bf8f7be/framework/wazuh/core/utils.py#L1225-L1228

And `{}` need to be added to the 3 main groups